internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -1008,6 +1028,20 @@ func defaultMockLogManager() *testLogManager {
 	}
 }
 
+// defaultMockFiltererV2 provides a default mock V2 filterer to be used
+// throughout the tests. Tests can override these defaults by calling
+// defaultMockFiltererV2, updating fields as necessary on the returned
+// *testFiltererV2, and then setting rpcTest.mockFiltererV2 as that
+// *testFiltererV2.
+func defaultMockFiltererV2() *testFiltererV2 {
+	block432100Filter := hexToBytes("11cdaad289eb092b5fd6ad60c7f7f197c2234dcbc74" +
+		"b14e1a477b319eae9d189cfae45f06a225965c7e932fc7600")
+	filter, _ := gcs.FromBytesV2(20, 1<<20, block432100Filter)
```

Comment:
It probably doesn't matter too much given these are tests, but this really should be using `blockcf2.B, blockcf2.M` since this is the actual v2 filter data for block 432100 those are the real parameters that need to be used to use the filter properly.

If you attempted to match against anything that is actually in the filter, it would fail due to the improper parameters.

Reply:
For reference, they are defined in the relevant DCP0005 [here](https://github.com/decred/dcps/blob/master/dcp-0005/dcp-0005.mediawiki#version-2-block-filters).

Reply:
Thanks, missed updating that from the previous empty default.  I prefer the test to be as close to reality as possible as well, so updated.  The DCP was very helpful to read through as well.